week-3/F-array-forEach/exercise.js

```diff
@@ -8,7 +8,19 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-
+function multiples(num) {
```

Comment:
I had to do this for my first real job interview 😆

Reply:
oh really! how did you get on with it? did you manage to secure the job?